pkgs/data/fonts/hubot-sans/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "hubot-sans";
+  version = "1.0";
+
+  src = fetchFromGitHub {
```

Comment:
I believe the procedure to install fonts is a bit different, but I need to look at the other expressions.

Reply:
I took some example from Inconsolata: https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/inconsolata/default.nix

Reply:
I will not complain, however I prefer this.

https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/data/fonts/b612/default.nix

Reply:
I tried, but for some reason I couldn't make it working. Had a conversation about that with Dee on Matrix:

![image](https://user-images.githubusercontent.com/252042/201469415-1ee6d788-c463-466d-914f-bf9d93aa870d.png)

Reply:
Feel free to let me know what to do to fix and I'll do the PR.

Reply:
@DeeUnderscore ^^

Reply:
Same issue with `b612` font: https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/data/fonts/b612/default.nix#L6